config/locales/in_person_proofing/en.yml

```diff
@@ -88,6 +88,7 @@ en:
       barcode: You’re ready to verify your identity in person
       location: Select a location to verify your ID
       prepare: Verify your identity in person
+      reminder: You have %{days_remaining} days left to verify your identity in person
```

Comment:
Since this is only used in the user_mailer I suggest we move it to `config/locales/user_mailer/en.yml` and similar for es/fr